src/sentry/api/endpoints/project_releases.py

```diff
@@ -53,7 +56,7 @@ class ReleaseSerializer(serializers.Serializer):
 
     def validate_version(self, attrs, source):
         value = attrs[source]
-        if not set(value).isdisjoint(set(string.whitespace)):
```

Comment:
I am not sure what you mean ... But it was a strange way to detect if there is a `string.whitespace` in the `value` var.